test/e2e/gameserver_test.go

```diff
@@ -183,19 +183,21 @@ func TestUnhealthyGameServersWithoutFreePorts(t *testing.T) {
 	t.Parallel()
 	nodes, err := framework.KubeClient.CoreV1().Nodes().List(metav1.ListOptions{})
 	assert.Nil(t, err)
-
-	// gate
 	assert.True(t, len(nodes.Items) > 0)
 
 	gs := framework.DefaultGameServer(defaultNs)
-	gs.Spec.Ports[0].HostPort = 7515
+	// choose port out of the minport/maxport range
+	gs.Spec.Ports[0].HostPort = 5555
```

Comment:
Also smart, no collisions.

Reply:
Added description of the root cause in original ticket. https://github.com/googleforgames/agones/issues/1376#issuecomment-614575113